.travis/run-verilator.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+export PATH=$HOME/tools/bin:$PATH
+
+verilator --lint-only rtl/verilog/*.v +incdir+rtl/verilog
```

Comment:
I see verilator had failed tring to find FPU modules. It looks like the line 5 should additionally contain `rtl/verilog/pfpu/*.v` inoptions list.

Reply:
Hello, yes.  Sorry, I meant to reply yesterday but some problem meant my reply was not sent.  See more details below.